workdir/openapi.yaml

```diff
@@ -18453,6 +18478,73 @@ paths:
     parameters:
     - $ref: '#/components/parameters/org_id'
     - $ref: '#/components/parameters/sso_id'
+  /api/v1/orgs/{org_id}/ssr/export_idtokens:
+    parameters:
+    - $ref: '#/components/parameters/org_id'
+    post:
+      description: While using Mist to onboard 128T devices, you can export the IDTokens
```

Comment:
description: While using Mist to onboard 128T devices, you can export the IDTokens
        and import it into Conductor so it will only allow those devices to get securely

Alternative:

      description: Export IDTokens from Mist to import into Conductor to securely allow SSR devices during onboarding.